yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -675,7 +673,21 @@ export class Sequencer extends (EventEmitter as new () => TypedEventEmitter<Sequ
         this.log.verbose(`Collected ${attestations.length} attestations`, { blockHash, blockNumber });
       }
 
-      await this.enqueuePublishL2Block(block, attestations, txHashes, invalidateBlock, publisher);
+      const attestationsAndSigners = new CommitteeAttestationsAndSigners(attestations ?? []);
+      const attestationsAndSignersSignature = this.validatorClient
+        ? await this.validatorClient.signAttestationsAndSigners(
+            attestationsAndSigners,
+            proposerAddress ?? publisher.getSenderAddress(),
+          )
```

Comment:
I'm not sure if the fallback on the sender address is correct. Shouldn't we just use an empty signature if there is no proposer defined for this iteration?

Reply:
Should be able to just give it an empty ye.